lib/spatial_features/importers/base.rb

```diff
@@ -5,10 +5,11 @@ module Importers
     class Base
       attr_reader :errors
 
-      def initialize(data, make_valid: false)
+      def initialize(data, make_valid: false, tmpdir: nil)
         @make_valid = make_valid
         @data = data
         @errors = []
+        @tmpdir = tmpdir
```

Comment:
This ivar is never actually accessed but this follows the pattern of taking all the initializer params from the subclasses.